scripts/run_app.sh

```diff
@@ -13,6 +13,7 @@ export DM_API_AUTH_TOKEN=${DM_API_AUTH_TOKEN:=myToken}
 
 export DM_SUBMISSIONS_BUCKET=${DM_SUBMISSIONS_BUCKET:=digitalmarketplace-documents-dev-dev}
 export DM_COMMUNICATIONS_BUCKET=${DM_COMMUNICATIONS_BUCKET:=digitalmarketplace-documents-dev-dev}
+export DM_AGREEMENTS_BUCKET=${DM_AGREEMENTS_BUCKET:=digitalmarketplace-documents-dev-dev}
```

Comment:
I haven't touched this in a while, but I think we used `dev-dev` because S3 uploader expected a bucket name format it could parse. If that changed we could probably use something different.
I don't think the name matters that much, but we should just choose one (or few, if we have different use cases) and delete all other dev buckets from S3 at some point.